src/app.js

```diff
@@ -1,3 +1,15 @@
 'use strict';
 
-// Write your code here
+const { checkIsValidUserInput } = require('./modules/checkIsValidUserInput');
+const { generateRandomNumber } = require('./modules/generateRandomNumber');
+const { getBullsAndCows } = require('./modules/getBullsAndCows');
+
+const numberToGuess = generateRandomNumber();
+const userInput = process.argv.slice(2).join('');
+
+// eslint-disable-next-line no-console
+console.log(userInput);
+
+if (checkIsValidUserInput(userInput)) {
+  getBullsAndCows(userInput, numberToGuess);
```

Comment:
The `getBullsAndCows` function expects `userInput` to be a number, but it is currently a string. Consider converting `userInput` to a number before passing it to the function, for example using `parseInt(userInput, 10)`.